TaskOne/demo.js

```diff
@@ -0,0 +1,39 @@
+const genhash = require("./hash.js");
+const args = process.argv.slice(2);
+path = args[0];
+const fs = require("fs");
```

Comment:
as a good practice, keep all the imports (require statements together on top)
and a leave space just for readability